colors/dracula.vim

```diff
@@ -81,8 +81,8 @@ if !exists('g:dracula_underline')
   let g:dracula_underline = 1
 endif
 
-if !exists('g:dracula_undercurl') && g:dracula_underline != 0
-  let g:dracula_undercurl = 1
+if !exists('g:dracula_undercurl')
+  let g:dracula_undercurl = g:dracula_underline
```

Comment:
LGTM. @dsifford ?

Reply:
Yep!

Reply:
Thanks for your review and explanation. Cheers!